content/en/docs/reference/config/analysis/ist0101/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: ReferencedResourceNotFound
+layout: analysis-message
+---
+
+This message occurs when your Configuration Resource Definition (CRD) file
```

Comment:
CRD defines the schema for the resource, not the resource itself. So this should be worded differently - probably "istio config" or something similar, not sure how we word this in other parts of the docs

Reply:
fixed